homeassistant/components/alarm_control_panel/lupusec.py

```diff
@@ -0,0 +1,90 @@
+"""
+This component provides HA alarm_control_panel support for Lupusec System.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.lupusec/
+"""
+
+from datetime import timedelta
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanel
+from homeassistant.components.lupusec import LupusecDevice
+
+import homeassistant.util.dt as dt_util
+
+from homeassistant.const import (STATE_ALARM_ARMED_AWAY,
+                                 STATE_ALARM_ARMED_HOME,
+                                 STATE_ALARM_DISARMED)
+
+DEPENDENCIES = ['lupusec']
+DOMAIN = 'lupusec'
+ICON = 'mdi:security'
+
+SCAN_INTERVAL = timedelta(seconds=2)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up an alarm control panel for a Lupusec device."""
+    data = hass.data[DOMAIN]
+
+    alarm_devices = [LupusecAlarm(data, data.lupusec.get_alarm(), data.name)]
+
+    data.devices.extend(alarm_devices)
```

Comment:
Don't store entities in a shared data container. Entities should not be accessed directly outside their platforms or entity helpers. If we need to communicate from component to platform (entity) we can use our dispatch helper.

Reply:
I would need some help with that. I just copied the work done in the abode platform.

Reply:
This storage doesn't seem needed anyway, so we can probably just remove it.